### PR TITLE
Tests for the SDK draft

### PR DESCRIPTION
**Pull request by javiermtorres**
Adding tests to the draft

### CODE REVIEW

File: lumigator/python/mzai/backend/backend/tests/BUILD.pants [hunk 1]
```diff
@@ -6,4 +6,8 @@ python_sources(
 python_tests(
     name="backend_tests",
     sources=["**/test_*.py"],
+    dependencies=[
```
Comment (veekaybee) [inline lumigator/python/mzai/backend/backend/tests/BUILD.pants:9]: we don't need this

File: lumigator/python/mzai/backend/schemas/deployments.py [hunk 2]
```diff
@@ -28,3 +28,7 @@ class DeploymentEvent(BaseModel):
     deployment_type: DeploymentType
     status: DeploymentStatus
     detail: str | None = None
+
+
+class DeploymentEventList(RootModel):
```
Comment (veekaybee) [inline lumigator/python/mzai/backend/schemas/deployments.py:33]: we should only need to inherit from `BaseModel `. https://docs.pydantic.dev/latest/concepts/models/#model-methods-and-properties

File: lumigator/python/mzai/sdk/tests/data/job.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
+{
```
Comment (veekaybee) [inline lumigator/python/mzai/sdk/tests/data/job.json:1]: let's add these to a `static` resources folder like in evaluator to keep things consistent in the module https://github.com/mozilla-ai/lumigator/tree/main/lumigator/python/mzai/evaluator/tests/resources

Comment (javiermtorres) [reply lumigator/python/mzai/sdk/tests/data/job.json:1]: Is there anything else needed apart from changing the name of the directory?

File: lumigator/python/mzai/backend/schemas/deployments.py (hunk 2 above)
Comment (javiermtorres) [reply lumigator/python/mzai/backend/schemas/deployments.py:33]: Ok, thanks. This was an attempt at loading lists of X, until I noticed https://github.com/mozilla-ai/lumigator/blob/b1971690e0b9abf45280c69831f141be7164d468/lumigator/python/mzai/backend/schemas/extras.py#L21

File: lumigator/python/mzai/backend/backend/tests/BUILD.pants (hunk 1 above)
Comment (javiermtorres) [reply lumigator/python/mzai/backend/backend/tests/BUILD.pants:9]: Let me recheck. The tests were not finding some source code before adding these dependencies. But maybe it's not the right way to make pytest find them.

Comment (javiermtorres) [reply lumigator/python/mzai/backend/backend/tests/BUILD.pants:9]: Ok, I still have issues with these tests but they don't seem the previous errors complaining that schemas could not be resolved. AFAICT this is not present in the new SDK PRs.

**Comment by javiermtorres** [comment]
Closed to be split up in several PRs addressing specific endpoints

**Pull request closed without merging**